src/CMakeLists.txt

```diff
@@ -302,3 +310,16 @@ endif()
 if(BUILD_TESTS)
   add_subdirectory(test)
 endif()
+
+
+install(TARGETS ${installable_targets}
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+)
+unset(installable_targets)
+
+if(INSTALL_MAN)
+  install(DIRECTORY ../doc/man/
```

Comment:
I'm not really familiar with how our manfiles are handled, but it seems strange to me that we'd install the placeholders rather than files generated at install-time by `gen-manpages.py`

Presumably the answer comes from the commit message in daf1ebf0b1c2020169d75fc52c3be87207693a04: "We can't remove them completely because `make dist` relies on the files being present."

So now that we won't have that restriction anymore, do we need these placeholders?

Reply:
> I'm not really familiar with how our manfiles are handled, but it seems strange to me that we'd install the placeholders rather than files generated at install-time by `gen-manpages.py`

This branch mimics the Autotoos-based build system in master right now: https://github.com/hebasto/bitcoin/blob/master/doc/man/Makefile.am

> Presumably the answer comes from the commit message in [daf1ebf](https://github.com/hebasto/bitcoin/commit/daf1ebf0b1c2020169d75fc52c3be87207693a04): "We can't remove them completely because `make dist` relies on the files being present."
> 
> So now that we won't have that restriction anymore, do we need these placeholders?

That would be possible after migration to CMake, right?

Reply:
Yeah, I guess this is just copying our current behavior. It's just that our current behavior is only needed (afact) because of autotools.

Maybe just add: "TODO: these stubs are no longer needed. man pages should be generated at install time."

Reply:
> Yeah, I guess this is just copying our current behavior. It's just that our current behavior is only needed (afact) because of autotools.
> 
> Maybe just add: "TODO: these stubs are no longer needed. man pages should be generated at install time."

Thanks! Done.